### PR TITLE
Create simple REST API for user, add postman collection

### PR DESCRIPTION
**Pull request by alialo2001**
Hello dear mentor! I have created a simple REST API for "user" clients. The app is simple and not connected to a database, but has needed endpoints for users. Please let me know if I need to extend the app at this stage. Thank you and have a wonderful day.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "budgetify",
+  "version": "1.0.0",
+  "description": "��#\u0000 \u0000B\u0000u\u0000d\u0000g\u0000e\u0000t\u0000i\u0000f\u0000y\u0000\r\u0000 \u0000\r\u0000 \u0000#\u0000#\u0000 \u0000T\u0000h\u0000e\u0000 \u0000a\u0000p\u0000p\u0000 \u0000i\u0000s\u0000 \u0000d\u0000e\u0000s\u0000i\u0000g\u0000n\u0000e\u0000d\u0000 \u0000f\u0000o\u0000r\u0000 \u0000a\u0000d\u0000u\u0000l\u0000t\u0000s\u0000 \u0000t\u0000o\u0000 \u0000t\u0000r\u0000a\u0000c\u0000k\u0000 \u0000t\u0000h\u0000e\u0000i\u0000r\u0000 \u0000f\u0000i\u0000n\u0000a\u0000n\u0000c\u0000i\u0000a\u0000l\u0000 \u0000e\u0000x\u0000p\u0000e\u0000n\u0000d\u0000i\u0000t\u0000u\u0000r\u0000e\u0000s\u0000\r\u0000 \u0000\r\u0000 \u0000#\u0000#\u0000#\u0000#\u0000 \u0000T\u0000E\u0000A\u0000M\u0000[\u00005\u0000]\u0000",
```
Comment (AudriusBaltusis) [inline package.json:4]: Please fix description

File: routes/user.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        amount: 2.1,
+        description: "Transportation"
+    },
+    {
+        amount: 14.0,
+        description: "Netflix subscription"
+    }
+]
+
+router.route('/expenses')
```
Comment (AudriusBaltusis) [inline routes/user.js:25]: API looks incomplete. At least you should create CRUD for `users`, `expenses`, `incomes`, `categories` and so on.

**Review by AudriusBaltusis — CHANGES_REQUESTED** [review]

**Comment by alialo2001** [comment]
I extended the REST API. 

I plan to perform refactoring for the next homework.

File: public/js/user_repository.js [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+  // create functions
+  createIncome(userId, income, callback) {
+    const user = this.getUserById(userId);
+    income.id = this.setIncomeId(user);
+    user.incomes.push(income);
+    user.balance += income.amount;
+    this.updateDB(callback);
+  }
+
+  createExpense(userId, expense, callback) {
```
Comment (AudriusBaltusis) [inline public/js/user_repository.js:28]: Please create separate modules for `income`, `incomeCategory`, `user` and so on

**Review by AudriusBaltusis — APPROVED** [review]
Some refactoring is required. Please do it in second iteration

File: public/js/user_repository.js (hunk 3 above)
Comment (alialo2001) [reply public/js/user_repository.js:28]: Got it. Thank you!

**Pull request merged**